www/include/home/customViews/views.php

```diff
@@ -46,6 +46,8 @@
 
 try {
 
+    require_once _CENTREON_PATH_ . "vendor/smarty/smarty/libs/Smarty.class.php";
```

Comment:
Is it required to fix pagination ?

Reply:
no I guess it went over from the merge, I guess after including bootstrap it is not needed, so i will remove it